plugins/modules/packaging/os/pkgin.py

```diff
@@ -126,7 +131,8 @@ def query_package(module, name):
     Possible return values:
     * "present"  - installed, no upgrade needed
     * "outdated" - installed, but can be upgraded
-    * False      - not installed or not found
+    * False      - not installed
```

Comment:
Wondering whether it would be cleaner if the return value would be `absent` and not `False`.

End instead maybe the return values should become enums instead a mix of strings and a bool

Reply:
Definitely. It would be good to stick to one type (or two, with the second being `None`).

Reply:
Introduced an enum flag. I am just wondering whether some tests running against Python < 3.6 will fail.

Reply:
Changed `Flag` to `IntFlag` since tests are running for Python versions >= 2.6